lib/creek/styles/value_convert/base.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Creek
+  class Styles
+    module ValueConvert
+      class Base
+
+        def is_support?(value, type, style, options)
+          false
```

Comment:
Why false by default? It makes no sense, so probably it should be `raise NotImplementedError` as well

Reply:
No reason.  I did this like you say